perftest/spreadtestnd.cpp

```diff
@@ -2,25 +2,20 @@
 #include <finufft/spreadinterp.h>
 #include <finufft/test_defs.h>
```

Comment:
It seems like adding ``using namespace std::printf``  etc to ``test_defs.h`` would make this a whole lot less ugly...